deployment/artifacts/lynx.json

```diff
@@ -1,7057 +1,7064 @@
 {
-    "11155111": {
```

Comment:
There seems to be an issue here with the number of spaces for tabs, which is producing a huge diff. I'd prefer if the same indentation convention is preserved, so change history is better understood.

Reply:
I think this was `pre-commit` job kicking in. 

Initially, our registries weren't subject to this linter update from pre-commit, and then once the pre-commit linter job kicked in, we didn't want to have a massive diff, and so we continued to ignore the linter update.

Some options:
- Modify the pre-commit linter ignore these files
- Update the file at some point (now or later) based on the linter, and continue to let the linter ensure the format remains the same. If we update to the linter's format, we should probably also modify this line - https://github.com/nucypher/nucypher-contracts/blob/main/deployment/registry.py#L20 - to use indent 2.
...

Reply:
Side note, if we go with keeping the original format, @vzotova you can use the `normalize_registry` script to revert the file back to what it was before the linter kicked in.

Reply:
✔️

Reply:
Filed https://github.com/nucypher/nucypher-contracts/issues/319